### ISSUE TITLE
UX/accessibility Issue - Cannot reach the User Consent switches with tab on My Account

### ISSUE THREAD

**Issue by PereViki** [topcoder-platform/community-app#1401]
**Describe the bug**
All affordances should be available through tabbing through the site, however, the User Consent  switch cannot be reached from the keyboard.

**To Reproduce**
1. Go to 'https://www.topcoder-dev.com/settings/account'
2. Click on 'My Account'
3. Tab through page

**Expected behavior**
The user should be able to set the User Consent from the keyboard.

**Screenshots**
http://www.screencast.com/t/jgRYUTeq5

**Desktop: MacBook Pro**
 - OS: Mac OS High Sierra
 - Browser: Chrome
 - Version: 69.0.3497.92

@ThomasKranitsas
